src/YQWizard.cc

```diff
@@ -853,16 +853,24 @@ QLayout *YQWizard::layoutButtonBox( QWidget * parent )
     hbox->setMargin( 0 );
 
     // Help button
-    // QT handles duplicate shortcuts, it can be kept (bnc#880983)
+    // Qt handles duplicate shortcuts, it can be kept (bnc#880983)
     _helpButton = new YQWizardButton( this, parent, _( "&Help" ).toStdString());
     YUI_CHECK_NEW( _helpButton );
-    _helpButton->setShortcut( Qt::Key_F1 );
 
     connect( _helpButton, &pclass(_helpButton)::clicked,
 	     this,	  &pclass(this)::showHelp );
 
     hbox->addWidget( (QWidget *) _helpButton->widgetRep() );
 
+    // Help action to be able to react to F1 and Alt-H (bnc#973389)
+    _helpAction = new QAction( this );
+    _helpAction->setShortcut( Qt::Key_F1 );
+    addAction( _helpAction );
+
+    connect( _helpAction, SIGNAL( triggered() ),
+             this,        SLOT  ( showHelp()  ) );
```

Comment:
Since Qt 5 signals and slots can be checked at compile time already, so this can be written as 
```cxx
    connect( _helpAction, QAction::triggered,
             this,        YQWizard::showHelp  );
```

A few lines above I used a [pclass macro](https://github.com/libyui/libyui-qt/blob/89ee421b553dcf1f346bfb009b90c75b68db1802/src/YQUI.h#L43) to deduce the classes automatically as I was doing a mass replacement.